src/DynamoUtilities/CLIWrapper.cs

```diff
@@ -162,6 +165,47 @@ protected virtual string GetData(int timeoutms, Func<string> mockReadLine = null
             return completedTask == readStdOutTask ? readStdOutTask.Result : string.Empty;
         }
 
+        private void DecodeHTMLContent(ref string htmlContent)
```

Comment:
These two functions maybe better belong to `Md2Html` class?

Reply:
The two methods were moved to the Md2Html class but I had to do the methods static (because otherwise are not visible in CLIWrapper.cs)
https://github.com/DynamoDS/Dynamo/pull/15377/commits/33430d9cb836caa94d02429978d28021579d2319